api/v1/cluster_types.go

```diff
@@ -1554,11 +1583,13 @@ type RecoveryTarget struct {
 }
 
 // StorageConfiguration is the configuration of the storage of the PostgreSQL instances
+// TODO: update the comment
```

Comment:
I think we should handle this TODO before merging this PR

Reply:
hahahah, yes!